rts/Game/SelectedUnitsHandler.cpp

```diff
@@ -645,7 +645,7 @@ void CSelectedUnitsHandler::Draw()
 			bool myColor = true;
 			glColor4fv(cmdColors.buildBox);
 
-			for (const auto bi: unitHandler.GetBuilderCAIs()) {
+			for (const auto &bi: unitHandler.GetBuilderCAIs()) {
```

Comment:
can go w/o a reference here, because the contained pair of objects is just two pointers.

Reply:
Can you clarify? Is the performance the same?

Reply:
Should be the same in practice